### PR TITLE
[Feature] Introduce web socket for job submission

### PR DESCRIPTION
**Pull request by s7monk**
close: https://github.com/apache/paimon-webui/issues/439

### Purpose

Introduce websocket for job submission.

### Tests

JobControllerTest#testGetJobStatistics

### CODE REVIEW

File: paimon-web-server/src/main/java/org/apache/paimon/web/server/configrue/WebSocketConfig.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+import org.springframework.web.socket.config.annotation.WebSocketMessageBrokerConfigurer;
+
+/** Websocket config. */
+@Configuration
+@EnableWebSocketMessageBroker
+public class WebSocketConfig implements WebSocketMessageBrokerConfigurer {
+    @Bean
+    public TaskScheduler webSocketTaskScheduler() {
+        ThreadPoolTaskScheduler scheduler = new ThreadPoolTaskScheduler();
+        scheduler.setPoolSize(1);
```
Comment (SteNicholas) [inline paimon-web-server/src/main/java/org/apache/paimon/web/server/configrue/WebSocketConfig.java:37]: Could this pool size be configured?

File: paimon-web-server/src/main/java/org/apache/paimon/web/server/configrue/WebSocketConfig.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+    @Override
+    public void registerStompEndpoints(StompEndpointRegistry registry) {
+        registry.addEndpoint("/ws").setAllowedOrigins("*");
+    }
+
+    @Override
+    public void configureMessageBroker(MessageBrokerRegistry registry) {
+        registry.enableSimpleBroker("/topic")
+                .setHeartbeatValue(new long[] {30000, 30000})
```
Comment (SteNicholas) [inline paimon-web-server/src/main/java/org/apache/paimon/web/server/configrue/WebSocketConfig.java:51]: Ditto

File: paimon-web-server/src/main/java/org/apache/paimon/web/server/configrue/WebSocketConfig.java [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    }
+
+    @Override
+    public void registerStompEndpoints(StompEndpointRegistry registry) {
+        registry.addEndpoint("/ws").setAllowedOrigins("*");
+    }
+
+    @Override
+    public void configureMessageBroker(MessageBrokerRegistry registry) {
+        registry.enableSimpleBroker("/topic")
```
Comment (SteNicholas) [inline paimon-web-server/src/main/java/org/apache/paimon/web/server/configrue/WebSocketConfig.java:50]: Why does this introduce topic url?

Comment (s7monk) [reply paimon-web-server/src/main/java/org/apache/paimon/web/server/configrue/WebSocketConfig.java:61]: Here, a websocket target prefix is ​​configured. The front-end client will carry this prefix when listening to server messages.

**Pull request merged**